md/pages.md

```diff
@@ -1,150 +1,31 @@
 #  Pages
+Pages are resources intended to be used in applications. They are [added, exported, edited, and deleted](resource-management.md) as resources in Bonita Portal. 
 
-Page resources are intended for use in applications or to customize Bonita portal. See how to
-use it and make it available to users. 
-
-## Custom page usage examples
+## Provided pages 
+Bonita Portal comes with some provided pages to use in applications. They are wrapped pages developed in other technologies than Bonita UI Designer (AngularJS, React, PortalJS).  
+Starting with Bonita 2021.1, you can also download the Bonita User and Administrator Applications (from Bonita Studio Welcome page), and get more provided pages, developped with Bonita UI Designer.
 
+## Custom pages
+To create the perfect user experience for different profiles of applications users, you need to create custom pages. This can be done with Bonita UI Designer, on in your favorite IDE.
```

Comment:
```suggestion
To create the perfect user experience for different profiles of applications users, you need to create custom pages. This can be done with Bonita UI Designer, or in your favorite IDE.
```